packages/terra-consumer/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Badge
```

Comment:
You could just run the generator(https://github.com/cerner/generator-terra-module) if you just want a fresh package to start off with instead of having to remove all this badge stuff.